config/locales/translations/zendesk_apps_support.yml

```diff
@@ -204,3 +204,19 @@ parts:
       key: "txt.apps.admin.error.app_build.invalid_type_parameter.other"
       title: "App builder job: multiple invalid parameter types error"
       value: "%{invalid_types} are invalid parameter types."
+  - translation:
+      key: "txt.apps.admin.error.app_build.unacceptable_boolean"
+      title: "App builder job: this value needs to be either true or false, but something else was passed in"
+      value: "%{field} must be a Boolean value, got %{value}"
```

Comment:
there are several field names, but we won't translate them, they will stay in English. They should all be singular. Some examples of possible values for `field` are: `requirements_only single_install signed_urls private`. The values for `value` are all supplied by the user though.